solutions/instances/main.tf

```diff
@@ -51,6 +53,19 @@ locals {
   cloud_logs_target_name     = var.prefix != null ? "${var.prefix}-cloud-logs-target" : "cloud-logs-target"
   at_cos_route_name          = var.prefix != null ? "${var.prefix}-at-cos-route" : "at-cos-route"
   at_cloud_logs_route_name   = var.prefix != null ? "${var.prefix}-at-cloud-logs-route" : "at-cloud-logs-route"
+  metric_router_target_name  = var.prefix != null ? "${var.prefix}-cloud_monitoring_target" : "cloud_monitoring_target"
+  metric_router_route_name   = var.prefix != null ? "${var.prefix}-metric-routing-route" : "metric-routing-route"
+
+  default_metrics_router_route = [{
+    name = local.metric_router_route_name
+    rules = [{
+      action = "send"
+      targets = [{
+        id = module.observability_instance.metrics_router_targets[local.metric_router_target_name].id
+      }]
+      inclusion_filters = []
```

Comment:
Just for confirmation - 
- is it as per the recommendation you received?
- If user add additional routes, do they need to remove this default route and it can still be there?

Reply:
If user doesn't add a metrics routing route this `default_route` will be created which is sending all the metrics to the cloud monitoring target. 

If user inputs their own route, the current code creates that and this route will not be created.